docs/samples/shards/UI/Grid/1.edn

```diff
@@ -0,0 +1,32 @@
+(defloop main-wire
+  (GFX.MainWindow
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (UI.CentralPanel
+      (UI.Grid
+       :Striped true
+       :Contents
+       (->
+        "Col 1" (UI.Label)
+        "Col 2" (UI.Label)
+        "Col 3" (UI.Label)
+        (UI.RowEnd)
```

Comment:
I'd call it EndRow (or NextRow)

Reply:
I will choose `NextRow` since this creates empty rows:
```clj
(UI.Grid
 :Striped true
 :Contents
 (->
  "Col 1" (UI.Label)
  "Col 2" (UI.Label)
  "Col 3" (UI.Label)
  (UI.NextRow)
  (UI.NextRow)
  (UI.NextRow)
  "Hello" (UI.Label)
  "shards" (UI.Label)
  "!" (UI.Label)))
```